---
date: "2021-08-08T00:00:00Z"
external_link: ""
image:
  caption: Photo by Dr_Microbe on Getty Images
  focal_point: Smart
slides: #example
summary: #An example of using the in-built project page.
tags:
- NHGRI
- NIH
- Candida auris
- Computational biology
title: Functional Annotation of Candida auris Genomes Using Funannotate
url_code: ""
url_pdf: "content/project/NHGRI/NHGRI.pdf"
url_slides: ""
url_video: ""
---

In the [Segre Lab](https://www.genome.gov/staff/Julie-Segre-PhD) at the NHGRI, I worked with *Candida auris* nanopore and RNA sequences from a US hospital outbreak to develop an in-house annotation pipeline for the fungus's genome that utilizes [Funannotate](https://github.com/nextgenusfs/funannotate). Annotation of the genome is a critical step in finding pathogenic features and differences between clades/geographic outbreaks. I tested several training parameters and compared the outputs of my annotations to those made by NISC (NIH Intramural Sequencing Center) and available in NCBI's GenBank. This effort was made possible through invaluable guidance and support from [Diana Proctor](https://dmap02.github.io/personal-website/).

I presented my findings to the [Translational and Functional Genomics Branch](https://www.genome.gov/about-nhgri/Division-of-Intramural-Research/Translational-Functional-Genomics-Branch#:~:text=The%20Translational%20and%20Functional%20Genomics,metagenomic%20basis%20of%20human%20disorders) of the NHGRI and at the NIH summer poster session. 

- [Poster](NHGRI.pdf)
- [Annotation Code - Github](https://github.com/couldbewoese/funannotate_script.git)

